lib/appsignal/integrations/redis.rb

```diff
@@ -3,12 +3,10 @@
 module Appsignal
   module Integrations
     module RedisIntegration
-      def process(commands, &block)
-        sanitized_commands = commands.map do |command, *args|
-          "#{command}#{" ?" * args.size}"
-        end.join("\n")
+      def write(command)
+        sanitized_command = command[0] == :eval ? command[1] : command[0].to_s
```

Comment:
For the example I didn't include the `?` in the command. I guess what we could do is:
- if command is eval, replace all `ARGV[N]` and `KEYS[N]` with `?`
- if it's not an eval, keep it like it was before: `"#{command}#{" ?" * args.size}"`

Do you think it makes sense?

Reply:
I kept the old format for the non script commands here. Implemented here https://github.com/appsignal/appsignal-ruby/pull/727/commits/2a3d68c87c823de2cbb05dc95b405c4538543324

Reply:
I'm considering keeping the `?` in for keys and arguments for the eval command, so you can see how many keys and arguments were really given to the eval method and to keep it more consistent.

@alanhala do you see a problem with that?

Reply:
How it would look with the question marks for arguments in an eval script: 

```ruby
client = Redis.new
client.eval("return { KEYS, ARGV }", ["k1", "k2"], ["a1", "a2"])
client.eval("return { KEYS, ARGV }", :keys => ["k1", "k2"], :argv => ["a1", "a2"])
```

<img width="674" alt="Screen Shot 2021-05-28 at 17 19 59" src="https://user-images.githubusercontent.com/282402/120006236-ff2e5600-bfd8-11eb-8e53-1a850e0a6280.png">

Reply:
I'm not sure that would look ok if you have a non one-liner script. But it's fine by me. I would only like to see the script that is called.

Reply:
How do you want to show it if it's a non one-liner script? Just by adding a new line and then the question marks?

Reply:
It would look something like this, with the question marks on the last line after the end of the eval script.
```
Multi line KEYS
statement ARGS
end ? ? ?
```